scheduler/kubernetes/csi.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Deploying Portworx with CSI support"
+keywords: csi, portworx, container, Kubernetes, storage, Docker, k8s, pv, persistent disk 
+sidebar: home_sidebar
+---
+
+* TOC
+{:toc}
+
+[CSI](https://kubernetes-csi.github.io/), or _Container Storage Interface_, is
+the new model for integrating storage system service with Kubernetes and other
+orchestration systems. Kubernetes has had support for CSI since 1.10 as beta.
+
+With CSI, Kubernetes gives storage drivers the opportunity to release on their
+schedule. This allows storage vendors to upgrade, update, and enhance their drivers
+without the need to update Kubernetes, maintaining a consistent, dependable,
+orchestration system.
+
+## Install
+
+### Install using the Portworx spec generator
+When installing Portworx through [install.portworx.com](https://install.portworx.com)
```

Comment:
Currently we don't expose the CSI option on the install.portworx.com page. It's hidden.
I think we can keep the docs same and just add the option on the install page. What do you think?